src/qt.ts

```diff
@@ -135,6 +142,9 @@ export class Qt {
     public async assistantFilename(): Promise<string> {
         let searchdirs = [];
         let filesnames = ["assistant" + tools.exeExtension(), "Assistant" + tools.exeExtension()];
+        if(process.platform == "linux" && !await tools.fileExists("/etc/apt")) {
+            filesnames = ["assistant-qt5" + tools.exeExtension(), "assistant-qt6" + tools.exeExtension()];
```

Comment:
* Any special reason why you also check `/etc/apt`?
* I would suggest to extend the `filesnames` variable, so that it contains the orignal value + the 2 new variants
  Just for the rare case that `assistant` will be just named `assistant`